Mathlib/Algebra/Category/ModuleCat/AB.lean

```diff
@@ -30,3 +32,13 @@ instance : AB4 (ModuleCat.{u} R) := AB4.of_AB5 _
 instance : AB4Star (ModuleCat.{u} R) where
   ofShape J :=
     HasExactLimitsOfShape.domain_of_functor (Discrete J) (forget₂ (ModuleCat R) AddCommGrp.{u})
+
+lemma ModuleCat.isSeparator : IsSeparator (ModuleCat.of R R) := fun X Y f g h ↦ by
+  simp only [Set.mem_singleton_iff, forall_eq, ModuleCat.hom_ext_iff, LinearMap.ext_iff] at h
+  ext x
+  simpa using h (ModuleCat.ofHom (LinearMap.toSpanSingleton R X x)) 1
+
+instance : HasSeparator (ModuleCat.{u} R) where
+  hasSeparator := ⟨ModuleCat.of R R, ModuleCat.isSeparator R⟩
```

Comment:
We can probably achieve this for `ModuleCat.{v} R` for `Small.{v} R`.

Reply:
done